src/ontology/uberon-edit.obo

```diff
@@ -227394,6 +227418,15 @@ is_a: UBERON:0001744 ! lymphoid tissue
 relationship: part_of UBERON:0010417 ! lymph node T cell domain
 created_by: http://orcid.org/0000-0002-2825-0621
 
+[Term]
+id: UBERON:8420000
+name: hair of scalp
+def: "Hair that grows on the scalp." [PMID:3126903]
```

Comment:
See above. Follow standard genus differentia definition patterns. e//g "a strand of hair that..."